editor/src/components/canvas/canvas-strategies/absolute-resize-helpers.ts

```diff
@@ -5,7 +5,7 @@ export const verticalPins: Array<AbsolutePin> = ['top', 'height', 'bottom']
 
 export function hasAtLeastTwoPinsPerSide(props: { [key: string]: any }): boolean {
   return (
-    horizontalPins.filter((pin) => props['style'][pin] != null).length >= 2 &&
-    verticalPins.filter((pin) => props['style'][pin] != null).length >= 2
+    horizontalPins.filter((pin) => props['style']?.[pin] != null).length >= 2 &&
```

Comment:
Is this clearer?
```suggestion
    horizontalPins.filter((pin) => props?.style?.pin != null).length >= 2 &&
```

Reply:
hmm that's no good because pin is a variable, but I can remove the brackets from around style
```suggestion
    horizontalPins.filter((pin) => props.style?.[pin] != null).length >= 2 &&
```

Reply:
pushed in bf62a4ec5

Reply:
> hmm that's no good because pin is a variable, but I can remove the brackets from around style

OMG, proving my own point that it made me blind to that being a variable.